docs/adr/adr-006-befp.md

```diff
@@ -0,0 +1,71 @@
+# ADR #006: Celestia-Node Bad encoding fraud proof(BEFP)
+
+## Changelog
+
+- 2021.03.03 - init commit
+
+## Authors
+
+@vgonkivs @Bidon15 @adlerjohn
+
+## Context
+
+In case when node receives errors `ErrByzantineRow`/`ErrByzantineCol` it should generate a fraud proof that broadcasts message to light clients and informs that block could be malicious.
```

Comment:
```suggestion
In case when node receive `ErrByzantineRow`/`ErrByzantineCol` from the [rsmt2d](https://github.com/celestiaorg/rsmt2d) library it should generate a fraud proof that gets broadcasted to light clients and informs that block could be malicious.
```